.github/workflows/ci-weekly.yml

```diff
@@ -152,7 +155,7 @@ jobs:
 
     - name: Install package dependencies
       run: yarn install --frozen-lockfile --ignore-scripts
-
+    
```

Comment:
```suggestion
```

Reply:
by design, as empty rows between steps/tasks makes it easier to read.